ElectronClient/app/gui/NotePropertiesDialog.jsx

```diff
@@ -347,7 +363,17 @@ class NotePropertiesDialog extends React.Component {
 		const formNote = this.state.formNote;
 
 		const buttonComps = [];
-		buttonComps.push(<button key="ok" style={styles.button} onClick={this.okButton_click}>{_('Apply')}</button>);
+		buttonComps.push(
+			<button
+				key="ok"
+				style={styles.button}
+				onClick={this.okButton_click}
+				ref="okButton"
```

Comment:
For refs, please use the new syntax React.createRef - https://reactjs.org/docs/refs-and-the-dom.html#creating-refs